app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -484,7 +482,7 @@ void setupGUI() {
         txtvPosition = findViewById(R.id.txtvPosition);
 
         SharedPreferences prefs = getSharedPreferences(PREFS, MODE_PRIVATE);
-        showTimeLeft = prefs.getBoolean(PREF_SHOW_TIME_LEFT, false);
+        showTimeLeft = UserPreferences.getShowRemainTimeSetting();
```

Comment:
The word "setting" sounds a bit redundant here. How about `isShowRemainingTime` or `shouldShowRemainingTime`?

Reply:
done